src/seedu/addressbook/Main.java

```diff
@@ -55,7 +56,8 @@ private void start(String[] launchArgs) {
             this.addressBook = storage.load();
             ui.showWelcomeMessage(VERSION, storage.getPath());
 
-        } catch (InvalidStorageFilePathException | StorageOperationException e) {
+        } catch (InvalidStorageFilePathException | StorageOperationException | FileNotFoundException e) {
+        	
```

Comment:
This works for the exercise but do consider the user's experience when handling exceptions.